tools/benchmark/binsize/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_benchmark_binsize"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[dependencies]
+serde_json = "1.0"
+clap = "2.33"
```

Comment:
question: ditto -- still being used for followup commits?

Reply:
Not at this time (0.3).